src/modules/deleteCompleted.js

```diff
@@ -1,17 +1,15 @@
-/* eslint-disable linebreak-style */
-/* eslint-disable import/no-cycle */
-/* eslint-disable no-plusplus */
+/* eslint-disable import/no-cycle, no-multi-assign */
 import renderTasks from '../index.js';
-import saveStorage from './saveStorage.js';
+import { saveStorage } from './saveStorage.js';
 
 const deleteCompleted = () => {
   const storedItems = JSON.parse(localStorage.getItem('todo-list'));
 
   const uncompletedItems = storedItems.filter((item) => item.completed === false);
 
-  let index = 1;
-  for (let i = 0; i <= uncompletedItems.length - 1; i++) {
-    uncompletedItems[i].index = index++;
+  let indx = 0;
+  for (let i = 0; i <= uncompletedItems.length - 1; i += 1) {
+    uncompletedItems[i].index = indx += 1;
```

Comment:
- [ ] While fixing the `plusplus` linter, you have included the new one which is `multi-assign`. Please fix this. I can see that you have used this sentence `uncompletedItems[i].index = indx += 1;` which is not a good code, instead you can use `uncompletedItems[i].index = indx + 1;`.